api/api/src/main/scala/uk/ac/wellcome/platform/api/Main.scala

```diff
@@ -29,7 +27,7 @@ object Main extends WellcomeTypesafeApp {
     val elasticClient = ElasticBuilder.buildElasticClient(config)
 
     val elasticConfig =
-      DisplayElasticConfig(indexV2 = Index(config.required("es.index.v2")))
+      ElasticBuilder.buildElasticConfig()
```

Comment:
Any reason to not hardcode it here? Something like 
```
 val elasticConfig = ElasticConfig( indexV2 = Index("v2-20200131"))	 
```
As @jamieparkinson was saying it is a bit hidden in `ElasticBuilder`. Also, it feels like it `ElasticBuilder` should remain fairly flexible if we're going down the route "we're hardcoding this only for the API"